backend/app/store/service/service_test.go

```diff
@@ -1279,7 +1277,9 @@ func TestService_submitImages(t *testing.T) {
 	lgr.Setup(lgr.Debug, lgr.CallerFile, lgr.CallerFunc)
 
 	mockStore := image.MockStore{}
-	mockStore.On("Commit", mock.Anything).Times(2).Return(nil)
+	mockStore.On("Commit", "dev/pic1.png").Once().Return(nil)
+	mockStore.On("Commit", "dev/pic2.png").Once().Return(nil)
+	mockStore.On("Commit", "cached_images/da39a3ee5e6b4b0d3255bfef95601890afd80709-da39a3ee5e6b4b0d3255bfef95601890afd80709").Times(2).Return(nil)
```

Comment:
worth to add mockStore.Assert to make sure all those Commits called

Reply:
We have `mockStore.AssertNumberOfCalls(t, "Commit", 3)` few lines lower, and all three mock calls have specified params so the call with something else will panic.
What I missed here is `Times(2)` instead of `Once()` which breaks this arithmetic, I'll fix that.

Reply:
Nvm, saw my error.